common/utils/inject_packet_loss_tc.yml

```diff
@@ -0,0 +1,33 @@
+---
+
+# This util induces packet drop in the pod provided. The inputs required are:
```

Comment:
Sorry for the nitpick. Either, we need to make this a separate util to 'just' do '100% packet loss on any given pod via tc' and place it under chaoslib/kubectl OR put the current util which does '100% packet loss on cstor pool' into chaoslib/openebs